8.frontend-react/managerapp/src/Components/Games/Games.component.jsx

```diff
@@ -0,0 +1,243 @@
+import axios from "axios";
+import React, { useEffect, useState } from "react";
+import Crud from "../CRUD/CRUD.component";
+import { RiDeleteBin5Fill } from "react-icons/ri";
+import { MdModeEdit } from "react-icons/md";
+import { updateGame, fetchGames, createGame } from "./scripts/serverRequests";
+import Loading from "../Loading/Loading";
+import Modal from "react-modal";
+import Form from "../Form/Form.component";
+import "./css/games-style.css";
+import "../../css/button-style.css"
+import '../../css/searchbox-style.css';
+
+
+function Games(props) {
+
+
+  const [inputValueStr,setInputValueStr]=useState([]);
+  const [open,setOpen]=useState(false);
+  const [games, setGames] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+
+
+
+  const addGame = async (e) => {
+    const gameName = e.target.elements.gameName.value.trim();
+    const genre = e.target.elements.genre.value.trim();
+    const rating = e.target.elements.rating.value.trim();
+    const price = e.target.elements.price.value.trim();
+    // const imgURL = e.target.elements.imgURL.value.trim();
+    const imgURL = e.target.elements.imgURL.files[0];
+    const formData = new FormData();
+    if(imgURL){
+    
+   
+    formData.append("image", imgURL, imgURL.name);
+  }
+    const arr = {
+      gameName: gameName,
+      genre: genre,
+      rating: rating,
+      price: price,
+      imgURL: imgURL.name?imgURL.name:"",
+    }
+  
+    try {
+      await axios.post("http://localhost:991/games/create/", arr);
+      if(imgURL)
+      await axios.post("http://localhost:991/games/saveimage/", formData);
+    } catch (err) {
+      throw err;
+    }
+  };
+  const modifyGame =  (e) => {
+    const gameName = e.target.elements.gameName.value.trim();
+    const genre = e.target.elements.genre.value.trim();
+    const rating = e.target.elements.rating.value.trim();
+    const price = e.target.elements.price.value.trim();
+    const imgURL = e.target.elements.imgURL.files[0];
+    const formData = new FormData();
+    if(imgURL){
+    formData.append("image", imgURL, imgURL.name);
+    }
+    const arr = {
+      id: inputValueStr.id,
+      gameName: gameName,
+      genre: genre,
+      rating: rating,
+      price: price,
+      imgURL: imgURL?imgURL.name:inputValueStr.imgURL?inputValueStr.imgURL:"",
+    }
+    if(imgURL)
+    axios.post("http://localhost:991/games/saveimage/", formData).then(res=>{
```

Comment:
move all URLS!!!!!!!!!!!!!!!!!!!!!!!!!!!!
to const file.

constants.js
import consts from constants.js
and use.


axios.post("http://localhost:991/games/saveimage/", formData).then(res=>{

to

axios.post(`${GAMES_ENDPOINT}`/saveimage/", formData).then(res=>{